fedora-toolbox

```diff
@@ -65,7 +73,17 @@ create()
 
         echo "$0: trying to configure working container $working_container_name" >&42
 
+        if $need_home_link ; then
+            if ! $prefix_sudo buildah run $working_container_name -- \
+                    sh -c 'rmdir /home && mkdir -m 0755 -p /var/home && ln -s var/home /home' 2>&42; then
+                $prefix_sudo buildah rm $working_container_name >/dev/null 2>&42
+                echo "$0: failed to make /home a symlink"
+                exit 1
+            fi
+        fi
+
         if ! $prefix_sudo buildah run $working_container_name -- useradd \
+                     --home-dir $HOME \
```

Comment:
This changes the value of `$HOME` to be `/var/home/user` instead of `/home/user`. I see that until Silverblue 29, `$HOME` is still pointing to the `/home/user` symbolic link on the host. Has that changed with newer `rpm-ostree` or Silverblue?

Reply:
Summarizing discussion with @owtaylor in #silverblue on Freenode:

It's better to retain the exact same `$HOME` string inside the toolbox as is on the host. Otherwise, the bash and zsh prompts get confused, because the current working directory no longer matches `$HOME` inside the toolbox. eg., `fedora-toolbox enter` inside the host's $HOME will lead to a confused bash prompt, and similarly $HOME/sub-directory for zsh.

This is relevant because even if `rpm-ostree` has switched the default in `/etc//default/useradd` to be `/var/home`, that change won't affect existing users, nor will it affect users created from Anaconda which might be using a different `/etc/default/useradd`.

Reply:
Also, the actual relevant line is not this, but this one further above:
```
HOME="$(readlink $HOME --canonicalize)"
```